components/assetcard/stories/assetcard.stories.js

```diff
@@ -0,0 +1,161 @@
+// Import the component markup template
+import { Template } from "./template";
+
+import { default as Checkbox } from "@spectrum-css/checkbox/stories/checkbox.stories.js";
+
+export default {
+  title: "Asset card",
+  description: "The asset card component allows users to select and manage assets and their metadata in a grid.",
+  component: "Assetcard",
+  argTypes: {
+    image: {
+      name: "Image",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      control: { type: "file", accept: ".svg,.png,.jpg,.jpeg,.webc" },
+    },
+    exampleImage: {
+      name: "Example images",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["landscape", "portrait", "square"],
+      control: "select",
+      if: { arg: 'image', truthy: false },
+    },
+    title: {
+      name: "Title",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Content",
+      },
+      control: { type: "text" },
+    },
+    headerContent: {
+      name: "Additional header content",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Content",
+      },
+      control: { type: "text" },
+    },
+    content: { table: { disable: true } },
+    selection: {
+      name: "Selection styles",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["checkbox", "highlight", "ordered"],
```

Comment:
I get an error on Storybook if I try to switch back and forth between these. Since you have the different variations set up as separate stories, I think it's ok that it doesn't work but I'm wondering if maybe we should remove the control if it isn't going to work? Just to avoid thinking you broke something down the line when it always functioned like this